pinot-core/src/main/java/com/linkedin/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -249,13 +250,13 @@ private boolean endCriteriaReached() {
     }
   }
 
-  private void handleTransientKafkaErrors(Exception e) throws  Exception {
+  private void handleTransientKafkaErrors(@Nonnull Exception e) throws Exception {
     consecutiveErrorCount++;
     if (consecutiveErrorCount > MAX_CONSECUTIVE_ERROR_COUNT) {
       segmentLogger.warn("Kafka transient exception when fetching messages, stopping consumption after {} attempts", consecutiveErrorCount, e);
       throw e;
     } else {
-      segmentLogger.warn("Kafka transient exception when fetching messages, retrying (count={})", consecutiveErrorCount, e);
+      segmentLogger.info("Kafka transient exception when fetching messages, retrying (count = {}, cause = {})", consecutiveErrorCount, e.toString());
```

Comment:
should be warn, but we don't need to log the exception stack.
ideally, we should log an INFO when we get back from the transient error, and the complete stack and error if we cannot.